Auto update failing for nushell

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [nushell](https://github.com/termux/termux-packages/blob/dcdb516068a4d7a7ee3707c054a613d6bd5d3243/packages/nushell) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating nushell [Current version: 0.89.0]
INFO: package being updated to 0.90.1.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2711k    0 2711k    0     0  5677k      0 --:--:-- --:--:-- --:--:-- 5677k

You are about to commit these changes:

--------------------
diff --git a/packages/nushell/build.sh b/packages/nushell/build.sh
index 0ee4a759c..5e4fd486b 100644
--- a/packages/nushell/build.sh
+++ b/packages/nushell/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.nushell.sh
 TERMUX_PKG_DESCRIPTION="A new type of shell operating on structured data"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.89.0"
+TERMUX_PKG_VERSION="0.90.1"
 TERMUX_PKG_SRCURL=https://github.com/nushell/nushell/archive/$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=1d7a7cc960bb8265214a7345ea3787a5bc22dcb7099d5c5056163df025c5527d
+TERMUX_PKG_SHA256=cb15559556311dea349a0f0b5fddeb3cc7a3adea9b0586753f0c632d69727084
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, zlib"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/nushell): 0.90.1


Not committing to Git!

INFO: Trying to build package.
INFO: Package nushell exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building nushell for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  36180      0 --:--:-- --:--:-- --:--:-- 36279
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5803      0 --:--:-- --:--:-- --:--:--  5825
gpg: Signature made Wed 07 Feb 2024 03:03:57 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  40079      0 --:--:-- --:--:-- --:--:-- 39996
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2474      0 --:--:-- --:--:-- --:--:--  2479
gpg: Signature made Sat 03 Feb 2024 09:00:29 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  89023      0 --:--:-- --:--:-- --:--:-- 89244
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3924      0 --:--:-- --:--:-- --:--:--  3910
100   833  100   833    0     0   3922      0 --:--:-- --:--:-- --:--:--  3910
gpg: Signature made Tue 06 Feb 2024 09:29:28 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency ca-certificates@1:2023.12.12 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2023.12.12 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.2.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.2.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/nushell/nushell/archive/0.90.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2711k    0 2711k    0     0  7255k      0 --:--:-- --:--:-- --:--:-- 7255k
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.75.0-x86_64-unknown-linux-gnu'

info: default toolchain set to '1.75.0-x86_64-unknown-linux-gnu'
  1.75.0-x86_64-unknown-linux-gnu unchanged - rustc 1.75.0 (82e1608df 2023-12-21)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: syncing channel updates for '1.73.0-x86_64-unknown-linux-gnu'
info: latest update on 2023-10-05, rust version 1.73.0 (cc66ad468 2023-10-03)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
/home/builder/termux-packages/packages/nushell/build.sh: line 27: [: missing `]'
    Updating crates.io index
 Downloading crates ...
  Downloaded dirs-sys-next v0.1.2
  Downloaded doc-comment v0.3.3
  Downloaded cassowary v0.3.0
  Downloaded minimal-lexical v0.2.1
  Downloaded crc v2.1.0
  Downloaded openssl-macros v0.1.1
  Downloaded libflate_lz77 v1.2.0
  Downloaded file-id v0.2.1
  Downloaded is-wsl v0.4.0
  Downloaded paste v1.0.14
  Downloaded tabled v0.14.0
  Downloaded polars-arrow v0.36.2
  Downloaded cssparser-macros v0.6.1
  Downloaded cssparser v0.31.2
  Downloaded plotters v0.3.5
  Downloaded ego-tree v0.6.2
  Downloaded base64 v0.13.1
  Downloaded hashbrown v0.13.2
  Downloaded simd-json v0.13.8
  Downloaded brotli-decompressor v2.5.1
  Downloaded brotli v3.4.0
  Downloaded libsqlite3-sys v0.26.0
  Downloaded libz-sys v1.1.15
  Downloaded uu_mktemp v0.0.23
  Downloaded human-date-parser v0.1.1
  Downloaded openssl-src v300.2.1+3.2.0
  Downloaded polars-json v0.36.2
  Downloaded polars-sql v0.36.2
  Downloaded gjson v0.8.1
  Downloaded libssh2-sys v0.3.0
  Downloaded libgit2-sys v0.16.1+1.7.1
  Downloaded async-stream-impl v0.3.5
  Downloaded async-stream v0.3.5
  Downloaded async-trait v0.1.77
  Downloaded parquet-format-safe v0.2.4
  Downloaded seq-macro v0.3.5
  Downloaded streaming-decompression v0.1.2
  Downloaded argminmax v0.6.1
  Downloaded env_logger v0.8.4
  Downloaded quickcheck v1.0.3
  Downloaded quickcheck_macros v1.0.0
  Downloaded smawk v0.3.2
  Downloaded crc32fast v1.3.2
  Downloaded adler v1.0.2
  Downloaded miniz_oxide v0.7.1
  Downloaded flate2 v1.0.28
  Downloaded pori v0.0.0
  Downloaded encoding_rs v0.8.33
  Downloaded anstyle v1.0.4
  Downloaded uuhelp_parser v0.0.24
  Downloaded uucore_procs v0.0.24
  Downloaded wax v0.6.0
  Downloaded str_indices v0.4.3
  Downloaded stable_deref_trait v1.2.0
  Downloaded bytesize v1.3.0
  Downloaded polars-lazy v0.36.2
  Downloaded sqlparser v0.39.0
  Downloaded miette-derive v5.10.0
  Downloaded git2 v0.18.1
  Downloaded oorandom v11.1.3
  Downloaded const_format_proc_macros v0.2.32
  Downloaded byte-unit v5.1.4
  Downloaded lz4-sys v1.9.4
  Downloaded derive_more v0.99.17
  Downloaded selectors v0.25.0
  Downloaded scraper v0.18.1
  Downloaded tiny-keccak v2.0.2
  Downloaded assert_cmd v2.0.13
  Downloaded anes v0.1.6
  Downloaded xmlparser v0.13.6
  Downloaded roxmltree v0.18.1
  Downloaded ureq v2.9.1
  Downloaded half v2.3.1
  Downloaded criterion v0.5.1
  Downloaded lexical-parse-float v0.8.5
  Downloaded fancy-regex v0.12.0
  Downloaded itertools v0.11.0
  Downloaded heapless v0.8.0
  Downloaded bincode v1.3.3
  Downloaded toml_edit v0.21.0
  Downloaded borsh v1.3.1
  Downloaded bitvec v1.0.1
  Downloaded ahash v0.7.7
  Downloaded hashbrown v0.12.3
  Downloaded rkyv v0.7.43
  Downloaded rust_decimal v1.33.1
  Downloaded textwrap v0.15.2
  Downloaded nix v0.27.1
  Downloaded enum_dispatch v0.3.12
  Downloaded polars-parquet v0.36.2
  Downloaded joinery v2.1.0
  Downloaded const_format v0.2.32
  Downloaded ropey v1.6.1
  Downloaded winnow v0.5.35
  Downloaded bytemuck v1.14.1
  Downloaded dtoa v1.0.9
  Downloaded dtoa-short v0.3.4
  Downloaded fxhash v0.2.1
  Downloaded servo_arc v0.3.0
  Downloaded charset v0.1.3
  Downloaded psm v0.1.21
  Downloaded stacker v0.1.15
  Downloaded rfc2047-decoder v1.0.2
  Downloaded eml-parser v0.1.4
  Downloaded ical v0.10.0
  Downloaded crunchy v0.2.2
  Downloaded const-random-macro v0.1.16
  Downloaded const-random v0.1.17
  Downloaded dlv-list v0.5.2
  Downloaded predicates-core v1.0.6
  Downloaded termtree v0.4.1
  Downloaded predicates-tree v1.0.9
  Downloaded wait-timeout v0.2.0
  Downloaded ciborium-io v0.2.2
  Downloaded ref-cast v1.0.22
  Downloaded float-cmp v0.9.0
  Downloaded titlecase v2.2.1
  Downloaded toml v0.8.8
  Downloaded umask v2.1.0
  Downloaded quick-error v2.0.1
  Downloaded supports-unicode v2.1.0
  Downloaded terminal_size v0.1.17
  Downloaded ciborium-ll v0.2.2
  Downloaded ciborium v0.2.2
  Downloaded tinytemplate v1.2.1
  Downloaded ctrlc v3.4.2
  Downloaded libmimalloc-sys v0.1.35
  Downloaded mimalloc v0.1.39
  Downloaded polars v0.36.2
  Downloaded memmap2 v0.7.1
  Downloaded polars-io v0.36.2
  Downloaded lexical-parse-integer v0.8.6
  Downloaded ref-cast-impl v1.0.22
  Downloaded uu_cp v0.0.23
  Downloaded uu_mkdir v0.0.23
  Downloaded fs_extra v1.3.0
  Downloaded portable-atomic v1.6.0
  Downloaded indicatif v0.17.7
  Downloaded uu_mv v0.0.23
  Downloaded uucore v0.0.24
  Downloaded bit-set v0.5.3
  Downloaded lru v0.12.1
  Downloaded uu_whoami v0.0.23
  Downloaded v_htmlescape v0.15.8
  Downloaded unicode-xid v0.2.4
  Downloaded rmp v0.8.12
  Downloaded proc-macro-crate v3.1.0
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded proc-macro-error v1.0.4
  Downloaded cfg_aliases v0.1.1
  Downloaded funty v2.0.0
  Downloaded radium v0.7.0
  Downloaded bytecheck_derive v0.6.11
  Downloaded simdutf8 v0.1.4
  Downloaded bytecheck v0.6.11
  Downloaded rend v0.4.1
  Downloaded rkyv_derive v0.7.43
  Downloaded seahash v4.1.0
  Downloaded uuid v1.7.0
  Downloaded owo-colors v3.5.0
  Downloaded is_ci v1.1.1
  Downloaded typed-arena v1.7.0
  Downloaded sxd-document v0.3.2
  Downloaded sxd-xpath v0.4.2
  Downloaded quoted_printable v0.5.0
  Downloaded ordered-multimap v0.7.1
  Downloaded difflib v0.4.0
  Downloaded predicates v3.1.0
  Downloaded plotters-backend v0.3.5
  Downloaded plotters-svg v0.3.5
  Downloaded polars-pipe v0.36.2
  Downloaded lexical-write-integer v0.8.5
  Downloaded lexical-core v0.8.5
  Downloaded halfbrown v0.2.4
  Downloaded value-trait v0.8.1
  Downloaded alloc-stdlib v0.2.2
  Downloaded atoi v2.0.0
  Downloaded now v0.1.3
  Downloaded unsafe-libyaml v0.2.10
  Downloaded serde_yaml v0.9.30
  Downloaded openssl-probe v0.1.5
  Downloaded native-tls v0.2.11
  Downloaded xattr v1.3.1
  Downloaded console v0.15.8
  Downloaded number_prefix v0.4.0
  Downloaded dunce v1.0.4
  Downloaded os_display v0.1.3
  Downloaded wild v2.2.1
  Downloaded hash32 v0.3.1
  Downloaded rmp-serde v1.1.2
  Downloaded linked-hash-map v0.5.6
  Downloaded syn_derive v0.1.8
  Downloaded borsh-derive v1.3.1
  Downloaded tap v1.0.1
  Downloaded wyz v0.5.1
  Downloaded ptr_meta_derive v0.1.4
  Downloaded ptr_meta v0.1.4
  Downloaded utf8-width v0.1.7
  Downloaded chrono-humanize v0.2.3
  Downloaded bit-vec v0.6.3
  Downloaded supports-color v2.1.0
  Downloaded supports-hyperlinks v2.1.0
  Downloaded is-terminal v0.4.10
  Downloaded unicode-linebreak v0.1.5
  Downloaded foreign_vec v0.1.0
  Downloaded lz4 v1.24.0
  Downloaded zstd-safe v7.0.0
  Downloaded libflate v1.4.0
  Downloaded rust-embed-utils v8.2.0
  Downloaded nom v7.1.3
  Downloaded vte v0.10.1
  Downloaded papergrid v0.10.0
  Downloaded num-bigint v0.2.6
  Downloaded yansi v0.5.1
  Downloaded polars-core v0.36.2
  Downloaded bytemuck_derive v1.5.0
  Downloaded libc v0.2.152
  Downloaded csv v1.3.0
  Downloaded polars-compute v0.36.2
  Downloaded polars-plan v0.36.2
  Downloaded array-init-cursor v0.2.0
  Downloaded adler32 v1.2.0
  Downloaded static_assertions v1.1.0
  Downloaded polars-utils v0.36.2
  Downloaded serde_repr v0.1.18
  Downloaded ethnum v1.5.0
  Downloaded planus v0.3.1
  Downloaded lsp-types v0.95.0
  Downloaded foreign-types-shared v0.1.1
  Downloaded miette v5.10.0
  Downloaded quick-error v1.2.3
  Downloaded peresil v0.3.0
  Downloaded chumsky v0.9.3
  Downloaded rust-ini v0.20.0
  Downloaded cast v0.3.0
  Downloaded criterion-plot v0.5.0
  Downloaded crossbeam-queue v0.3.11
  Downloaded lexical-write-float v0.8.5
  Downloaded lexical-util v0.8.5
  Downloaded alloc-no-stdlib v2.0.4
  Downloaded itertools v0.12.0
  Downloaded deranged v0.3.11
  Downloaded pest_meta v2.7.6
  Downloaded signal-hook v0.3.17
  Downloaded thread_local v1.1.7
  Downloaded signal-hook-registry v1.4.1
  Downloaded strip-ansi-escapes v0.2.0
  Downloaded shell-words v1.1.0
  Downloaded fallible-iterator v0.2.0
  Downloaded assert-json-diff v2.0.2
  Downloaded open v5.0.1
  Downloaded jobserver v0.1.27
  Downloaded md-5 v0.10.6
  Downloaded mockito v1.2.0
  Downloaded hashlink v0.8.4
  Downloaded rstest_macros v0.18.2
  Downloaded num-rational v0.4.1
  Downloaded print-positions v0.6.1
  Downloaded relative-path v1.9.2
  Downloaded rstest v0.18.2
  Downloaded num-bigint v0.4.4
  Downloaded comfy-table v7.1.0
  Downloaded multiversion-macros v0.7.3
  Downloaded multiversion v0.7.3
  Downloaded streaming-iterator v0.1.9
  Downloaded strength_reduce v0.2.4
  Downloaded zstd-sys v2.0.9+zstd.1.5.5
  Downloaded xxhash-rust v0.8.8
  Downloaded polars-time v0.36.2
  Downloaded arrow-format v0.8.1
  Downloaded dtparse v2.0.0
  Downloaded sysinfo v0.30.5
  Downloaded erased-serde v0.4.2
  Downloaded typetag-impl v0.2.15
  Downloaded procfs-core v0.16.0
  Downloaded procfs v0.16.0
  Downloaded openssl v0.10.63
  Downloaded iana-time-zone v0.1.59
  Downloaded pkg-config v0.3.29
  Downloaded serde_json v1.0.112
  Downloaded rle-decode-fast v1.0.3
  Downloaded avro-schema v0.3.0
  Downloaded polars-error v0.36.2
  Downloaded arrayvec v0.5.2
  Downloaded rustix v0.38.30
  Downloaded time-core v0.1.2
  Downloaded phf v0.11.2
  Downloaded filesize v0.2.0
  Downloaded shadow-rs v0.26.1
  Downloaded fd-lock v3.0.13
  Downloaded vte v0.11.1
  Downloaded os_pipe v1.1.5
  Downloaded vte_generate_state_changes v0.1.1
  Downloaded num-complex v0.4.4
  Downloaded is-docker v0.2.0
  Downloaded vcpkg v0.2.15
  Downloaded rusqlite v0.29.0
  Downloaded quick-xml v0.31.0
  Downloaded num v0.4.1
  Downloaded hex v0.4.3
  Downloaded unicode-width v0.1.11
  Downloaded unicode-segmentation v1.10.1
  Downloaded simplelog v0.12.1
  Downloaded semver v1.0.21
  Downloaded dashmap v5.5.3
  Downloaded futures-executor v0.3.30
  Downloaded zstd v0.13.0
  Downloaded polars-row v0.36.2
  Downloaded polars-ops v0.36.2
  Downloaded inventory v0.3.15
  Downloaded ansi-str v0.8.0
  Downloaded num-iter v0.1.43
  Downloaded heck v0.4.1
  Downloaded rustversion v1.0.14
  Downloaded futures-timer v3.0.2
  Downloaded strum_macros v0.25.3
  Downloaded strum v0.25.0
  Downloaded glob v0.3.1
  Downloaded rustc_version v0.4.0
  Downloaded indexmap v2.1.0
  Downloaded ahash v0.8.7
  Downloaded futures v0.3.30
  Downloaded dyn-clone v1.0.16
  Downloaded fast-float v0.2.0
  Downloaded phf_codegen v0.11.2
  Downloaded tokio v1.35.1
  Downloaded bytecount v0.6.7
  Downloaded dirs-next v2.0.0
  Downloaded num-format v0.4.4
  Downloaded crc-catalog v1.1.1
  Downloaded sqlparser v0.43.1
  Downloaded rust-embed-impl v8.2.0
  Downloaded is_executable v1.0.1
  Downloaded alphanumeric-sort v1.5.3
  Downloaded calamine v0.23.1
  Downloaded chardetng v0.1.17
  Downloaded csv-core v0.1.11
  Downloaded which v5.0.0
  Downloaded tempfile v3.9.0
  Downloaded snap v1.1.1
  Downloaded zerocopy v0.7.32
  Downloaded serial_test_derive v2.0.0
  Downloaded futures-io v0.3.30
  Downloaded target-features v0.1.5
  Downloaded rand_distr v0.4.3
  Downloaded is_debug v1.0.1
  Downloaded typetag v0.2.15
  Downloaded diff v0.1.13
  Downloaded ansitok v0.2.0
  Downloaded num-complex v0.2.4
  Downloaded hamcrest2 v0.3.0
  Downloaded num_threads v0.1.6
  Downloaded smartstring v1.0.1
  Downloaded rust-embed v8.2.0
  Downloaded indoc v2.0.4
  Downloaded lsp-server v0.7.6
  Downloaded lazy_static v1.4.0
  Downloaded termcolor v1.1.3
  Downloaded serial_test v2.0.0
  Downloaded atoi_simd v0.15.6
  Downloaded bracoxide v0.1.3
  Downloaded chrono-tz v0.8.5
  Downloaded ratatui v0.23.0
  Downloaded num-integer v0.1.45
  Downloaded num-rational v0.2.4
  Downloaded num v0.2.1
  Downloaded sys-locale v0.3.1
  Downloaded nu-ansi-term v0.50.0
  Downloaded time v0.3.31
  Downloaded fuzzy-matcher v0.3.7
  Downloaded pest v2.7.6
  Downloaded pretty_assertions v1.4.0
  Downloaded crossterm v0.27.0
  Downloaded pure-rust-locales v0.7.0
  Downloaded reedline v0.29.0
  Downloaded home v0.5.9
  Downloaded phf_shared v0.11.2
  Downloaded allocator-api2 v0.2.16
  Downloaded powerfmt v0.2.0
  Downloaded codepage v0.1.1
  Downloaded zip v0.6.6
  Downloaded parse-zoneinfo v0.3.0
  Downloaded chrono-tz-build v0.2.1
  Downloaded phf_generator v0.11.2
  Downloaded phf_macros v0.11.2
  Downloaded dialoguer v0.11.0
  Downloaded pest_generator v2.7.6
  Downloaded pest_derive v2.7.6
  Downloaded similar v2.4.0
  Downloaded notify-debouncer-full v0.3.1
  Downloaded lscolors v0.17.0
  Downloaded arrayvec v0.7.4
  Downloaded openssl-sys v0.9.99
  Downloaded arboard v3.3.0
  Downloaded libm v0.2.8
  Downloaded foreign-types v0.3.2
  Downloaded signal-hook-mio v0.2.3
  Downloaded fallible-streaming-iterator v0.1.9
  Downloaded time-macros v0.2.16
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.75.0-x86_64-unknown-linux-gnu'

info: default toolchain set to '1.75.0-x86_64-unknown-linux-gnu'
  1.75.0-x86_64-unknown-linux-gnu unchanged - rustc 1.75.0 (82e1608df 2023-12-21)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: component 'rust-std' for target 'aarch64-linux-android' is up to date
  Installing nu v0.90.1 (/home/builder/.termux-build/nushell/src)
    Updating crates.io index
warning: package `iana-time-zone v0.1.59` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
 Downloading crates ...
  Downloaded wayland-protocols v0.31.2
  Downloaded downcast-rs v1.2.0
  Downloaded wayland-scanner v0.31.1
  Downloaded tree_magic_mini v3.0.3
  Downloaded wayland-client v0.31.2
  Downloaded x11rb v0.12.0
  Downloaded x11rb-protocol v0.12.0
  Downloaded libloading v0.8.1
  Downloaded wl-clipboard-rs v0.8.0
  Downloaded derive-new v0.5.9
  Downloaded pwd v1.4.0
  Downloaded nix v0.26.4
  Downloaded wayland-sys v0.31.1
  Downloaded gethostname v0.3.0
  Downloaded wayland-protocols-wlr v0.2.0
  Downloaded wayland-backend v0.3.3
  Downloaded dlib v0.5.2
  Downloaded petgraph v0.6.4
  Downloaded memoffset v0.7.1
  Downloaded fixedbitset v0.4.2
  Downloaded trash v3.2.1
   Compiling proc-macro2 v1.0.78
   Compiling unicode-ident v1.0.12
   Compiling libc v0.2.152
   Compiling serde v1.0.196
   Compiling cfg-if v1.0.0
   Compiling version_check v0.9.4
   Compiling once_cell v1.19.0
   Compiling autocfg v1.1.0
   Compiling rustix v0.38.30
   Compiling quote v1.0.35
   Compiling linux-raw-sys v0.4.13
   Compiling syn v2.0.48
   Compiling utf8parse v0.2.1
   Compiling memchr v2.7.1
   Compiling errno v0.3.8
   Compiling getrandom v0.2.12
   Compiling num-traits v0.2.17
   Compiling unicode-width v0.1.11
   Compiling thiserror v1.0.56
   Compiling ahash v0.8.7
   Compiling log v0.4.20
   Compiling itoa v1.0.10
   Compiling zerocopy v0.7.32
   Compiling allocator-api2 v0.2.16
   Compiling hashbrown v0.14.3
   Compiling crossbeam-utils v0.8.19
   Compiling android_system_properties v0.1.5
   Compiling iana-time-zone v0.1.59
   Compiling vte_generate_state_changes v0.1.1
   Compiling pure-rust-locales v0.7.0
   Compiling android-tzdata v0.1.1
   Compiling unicase v2.7.0
   Compiling lazy_static v1.4.0
   Compiling either v1.9.0
   Compiling equivalent v1.0.1
   Compiling indexmap v2.1.0
   Compiling crc32fast v1.3.2
   Compiling ryu v1.0.16
   Compiling adler v1.0.2
   Compiling typenum v1.17.0
   Compiling miniz_oxide v0.7.1
   Compiling cc v1.0.83
   Compiling arrayvec v0.7.4
   Compiling serde_json v1.0.112
   Compiling flate2 v1.0.28
   Compiling crossbeam-epoch v0.9.18
   Compiling aho-corasick v1.1.2
   Compiling serde_derive v1.0.196
   Compiling thiserror-impl v1.0.56
   Compiling generic-array v0.14.7
   Compiling rayon-core v1.12.1
   Compiling regex-syntax v0.8.2
   Compiling nu-ansi-term v0.50.0
   Compiling crossbeam-deque v0.8.5
   Compiling vte v0.11.1
   Compiling regex-automata v0.4.5
   Compiling rust_decimal v1.33.1
   Compiling strip-ansi-escapes v0.2.0
   Compiling procfs v0.16.0
   Compiling hex v0.4.3
   Compiling rayon v1.8.1
   Compiling regex v1.10.3
   Compiling lscolors v0.17.0
   Compiling bitflags v2.4.2
   Compiling chrono v0.4.33
   Compiling nix v0.27.1
   Compiling procfs-core v0.16.0
   Compiling is-terminal v0.4.10
   Compiling num-format v0.4.4
   Compiling sys-locale v0.3.1
   Compiling dirs-sys-next v0.1.2
   Compiling bit-vec v0.6.3
   Compiling smawk v0.3.2
   Compiling pkg-config v0.3.29
   Compiling vcpkg v0.2.15
   Compiling unicode-linebreak v0.1.5
   Compiling is_ci v1.1.1
   Compiling supports-color v2.1.0
   Compiling textwrap v0.15.2
   Compiling nu-utils v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-utils)
   Compiling bit-set v0.5.3
   Compiling dirs-next v2.0.0
   Compiling supports-unicode v2.1.0
   Compiling supports-hyperlinks v2.1.0
   Compiling sysinfo v0.30.5
   Compiling erased-serde v0.4.2
   Compiling typetag-impl v0.2.15
   Compiling miette-derive v5.10.0
   Compiling terminal_size v0.1.17
   Compiling owo-colors v3.5.0
   Compiling utf8-width v0.1.7
   Compiling paste v1.0.14
   Compiling inventory v0.3.15
   Compiling typetag v0.2.15
   Compiling byte-unit v5.1.4
   Compiling miette v5.10.0
   Compiling nu-system v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-system)
   Compiling nu-path v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-path)
   Compiling fancy-regex v0.12.0
   Compiling chrono-humanize v0.2.3
   Compiling lru v0.12.1
   Compiling byteorder v1.5.0
   Compiling same-file v1.0.6
   Compiling walkdir v2.4.0
   Compiling nu-protocol v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-protocol)
   Compiling nu-glob v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-glob)
   Compiling smallvec v1.13.1
   Compiling rmp v0.8.12
   Compiling lock_api v0.4.11
   Compiling parking_lot_core v0.9.9
   Compiling rmp-serde v1.1.2
   Compiling terminal_size v0.3.0
   Compiling bincode v1.3.3
   Compiling itertools v0.12.0
   Compiling mio v0.8.10
   Compiling anstyle-parse v0.2.3
   Compiling colorchoice v1.0.0
   Compiling anstyle-query v1.0.2
   Compiling minimal-lexical v0.2.1
   Compiling scopeguard v1.2.0
   Compiling anstyle v1.0.4
   Compiling unicode-xid v0.2.4
   Compiling signal-hook v0.3.17
   Compiling const_format_proc_macros v0.2.32
   Compiling anstream v0.6.11
   Compiling nom v7.1.3
   Compiling signal-hook-registry v1.4.1
   Compiling strsim v0.10.0
   Compiling clap_lex v0.6.0
   Compiling bytesize v1.3.0
   Compiling siphasher v0.3.11
   Compiling rustversion v1.0.14
   Compiling time-core v0.1.2
   Compiling percent-encoding v2.3.1
   Compiling phf_shared v0.11.2
   Compiling clap_builder v4.4.18
   Compiling parking_lot v0.12.1
   Compiling openssl-sys v0.9.99
   Compiling linked-hash-map v0.5.6
   Compiling rand_core v0.6.4
   Compiling arrayvec v0.5.2
   Compiling tinyvec_macros v0.1.1
   Compiling uuhelp_parser v0.0.24
   Compiling uucore_procs v0.0.24
   Compiling nu-engine v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-engine)
   Compiling nu-plugin v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-plugin)
   Compiling nu-parser v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-parser)
   Compiling clap v4.4.18
   Compiling vte v0.10.1
   Compiling tinyvec v1.6.0
   Compiling rand v0.8.5
   Compiling nu-json v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-json)
   Compiling signal-hook-mio v0.2.3
   Compiling form_urlencoded v1.2.1
   Compiling block-buffer v0.10.4
   Compiling crypto-common v0.1.6
   Compiling libsqlite3-sys v0.26.0
   Compiling os_display v0.1.3
   Compiling dunce v1.0.4
   Compiling wild v2.2.1
   Compiling unicode-segmentation v1.10.1
   Compiling glob v0.3.1
   Compiling heck v0.4.1
   Compiling uucore v0.0.24
   Compiling strum_macros v0.25.3
   Compiling digest v0.10.7
   Compiling crossterm v0.27.0
   Compiling phf_generator v0.11.2
   Compiling unicode-normalization v0.1.22
   Compiling ansitok v0.2.0
   Compiling encoding_rs v0.8.33
   Compiling powerfmt v0.2.0
   Compiling ucd-trie v0.1.6
   Compiling unicode-bidi v0.3.15
   Compiling bitflags v1.3.2
   Compiling portable-atomic v1.6.0
   Compiling pest v2.7.6
   Compiling idna v0.5.0
   Compiling deranged v0.3.11
   Compiling parse-zoneinfo v0.3.0
   Compiling ansi-str v0.8.0
   Compiling phf_codegen v0.11.2
   Compiling strum v0.25.0
   Compiling nu-color-config v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-color-config)
   Compiling phf v0.11.2
   Compiling console v0.15.8
   Compiling crossbeam-channel v0.5.11
   Compiling hashlink v0.8.4
   Compiling cpufeatures v0.2.12
   Compiling thread_local v1.1.7
   Compiling num_threads v0.1.6
   Compiling fallible-streaming-iterator v0.1.9
   Compiling bytecount v0.6.7
   Compiling foreign-types-shared v0.1.1
   Compiling fnv v1.0.7
   Compiling fallible-iterator v0.2.0
   Compiling openssl v0.10.63
   Compiling pest_meta v2.7.6
   Compiling time v0.3.31
   Compiling sha2 v0.10.8
   Compiling papergrid v0.10.0
   Compiling foreign-types v0.3.2
   Compiling fuzzy-matcher v0.3.7
   Compiling chrono-tz-build v0.2.1
   Compiling url v2.5.0
   Compiling nu-cmd-base v0.90.1 (/home/builder/.termux-build/nushell/src/crates/nu-cmd-base)
   Compiling time-macros v0.2.16
   Compiling const_format v0.2.32
   Compiling openssl-macros v0.1.1
   Compiling itertools v0.11.0
   Compiling inotify-sys v0.1.5
   Compiling filetime v0.2.23
   Compiling number_prefix v0.4.0
   Compiling pathdiff v0.2.1
   Compiling v_htmlescape v0.15.8
   Compiling home v0.5.9
   Compiling ppv-lite86 v0.2.17
   Compiling native-tls v0.2.11
   Compiling is_debug v1.0.1
   Compiling shadow-rs v0.26.1
   Compiling rand_chacha v0.3.1
   Compiling which v5.0.0
   Compiling indicatif v0.17.7
   Compiling rusqlite v0.29.0
   Compiling inotify v0.9.6
   Compiling chrono-tz v0.8.5
   Compiling mime_guess v2.0.4
   Compiling tabled v0.14.0
   Compiling rust-embed-utils v8.2.0
   Compiling pest_generator v2.7.6
   Compiling fd-lock v3.0.13
   Compiling toml_datetime v0.6.5
   Compiling serde_spanned v0.6.5
   Compiling arboard v3.3.0
error[E0433]: failed to resolve: could not find `Clipboard` in `platform`
  --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:68:38
   |
68 |         Ok(Clipboard { platform: platform::Clipboard::new()? })
   |                                            ^^^^^^^^^ could not find `Clipboard` in `platform`

error[E0433]: failed to resolve: could not find `Clear` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:122:31
    |
122 |         Clear { platform: platform::Clear::new(&mut self.platform) }
    |                                     ^^^^^ could not find `Clear` in `platform`

error[E0433]: failed to resolve: could not find `Get` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:127:29
    |
127 |         Get { platform: platform::Get::new(&mut self.platform) }
    |                                   ^^^ could not find `Get` in `platform`

error[E0433]: failed to resolve: could not find `Set` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:132:29
    |
132 |         Set { platform: platform::Set::new(&mut self.platform) }
    |                                   ^^^ could not find `Set` in `platform`

error[E0412]: cannot find type `Clipboard` in module `platform`
  --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:62:33
   |
62 |     pub(crate) platform: platform::Clipboard,
   |                                    ^^^^^^^^^ not found in `platform`

error[E0412]: cannot find type `Get` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:139:33
    |
139 |     pub(crate) platform: platform::Get<'clipboard>,
    |                                    ^^^ not found in `platform`

error[E0412]: cannot find type `Set` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:163:33
    |
163 |     pub(crate) platform: platform::Set<'clipboard>,
    |                                    ^^^ not found in `platform`

error[E0412]: cannot find type `Clear` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.0/src/lib.rs:204:33
    |
204 |     pub(crate) platform: platform::Clear<'clipboard>,
    |                                    ^^^^^ not found in `platform`

Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `arboard` (lib) due to 8 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `nu v0.90.1 (/home/builder/.termux-build/nushell/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/nushell/src/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-02-07 06:17:24 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/7810629851">7810629851</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/nushell/build.sh
+++ b/packages/nushell/build.sh
@@ -2,13 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://www.nushell.sh
 TERMUX_PKG_DESCRIPTION="A new type of shell operating on structured data"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.89.0"
+TERMUX_PKG_VERSION="0.90.1"
 TERMUX_PKG_SRCURL=https://github.com/nushell/nushell/archive/$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=1d7a7cc960bb8265214a7345ea3787a5bc22dcb7099d5c5056163df025c5527d
+TERMUX_PKG_SHA256=cb15559556311dea349a0f0b5fddeb3cc7a3adea9b0586753f0c632d69727084
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, zlib"
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--features=extra"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS=("--no-default-features")
 
 termux_step_pre_configure() {
 	termux_setup_rust
@@ -24,9 +24,12 @@ termux_step_pre_configure() {
 		echo "INPUT(-l:libunwind.a)" >libgcc.so
 		popd
 	fi
-	if [ $TERMUX_ARCH != "i686" && $TERMUX_ARCH != "arm" ]; then
-		TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" --features=dataframe"
+
+	local _features="default-no-clipboard extra"
+	if [ $TERMUX_ARCH != "i686" ] && [ $TERMUX_ARCH != "arm" ]; then
+		_features+=" dataframe"
 	fi
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=("--features=$_features")
 
 	: "${CARGO_HOME:=$HOME/.cargo}"
 	export CARGO_HOME
@@ -40,6 +43,16 @@ termux_step_pre_configure() {
 		$_CARGO_TARGET_LIBDIR/libz.so.1
 	ln -sfT $(readlink -f $TERMUX_PREFIX/lib/libz.so.tmp) \
 		$_CARGO_TARGET_LIBDIR/libz.so
+}
+
+termux_step_make_install() {
+	cargo install \
+			--path . \
+			--jobs $TERMUX_MAKE_PROCESSES \
+			--no-track \
+			--target $CARGO_TARGET_NAME \
+			--root $TERMUX_PREFIX \
+			"${TERMUX_PKG_EXTRA_CONFIGURE_ARGS[@]}"
 }
 
 termux_step_post_make_install() {

